app/renderer/components/Inspector/shared.js

```diff
@@ -135,12 +140,10 @@ export const actionDefinitions = {
   },
   'Session': {
     'Session Capabilities': {
-      'Get Session Capabilities': {methodName: 'sessionCapabilities'}
+      'Get Session Capabilities': {methodName: 'getSession'}
```

Comment:
Does this work on web2driver? (just a question since this is not in w3c. I wondered if the client had this)

Reply:
with appium, web2driver includes all the old protocols as well, but the w3c versions are the ones that are used if there is a clash.